vars/managerPipeline.groovy

```diff
@@ -72,7 +72,7 @@ def call(Map pipelineParams) {
 
 
             string(defaultValue: '', description: '', name: 'scylla_ami_id')
-            string(defaultValue: "${pipelineParams.get('scylla_version', '5.1')}", description: '', name: 'scylla_version')
+            string(defaultValue: "${pipelineParams.get('scylla_version', '2022.2')}", description: '', name: 'scylla_version')
```

Comment:
why setting this default in the groovy, and not in our `jenkinsfile`?

Reply:
Cause then we have to change it in every every jenkinsfile. If we only set it here, we have a single setting to change every time.

Reply:
so, why are these https://github.com/scylladb/scylla-cluster-tests/pull/6164#discussion_r1199753238 and https://github.com/scylladb/scylla-cluster-tests/pull/6164#discussion_r1199753367 being set in the `jenkinsfile` and not using the default in this file?